geniza/templates/base.html

```diff
@@ -57,7 +57,7 @@
             <link rel="preload" as="font" type="font/woff" href="{{ FONT_URL_PREFIX }}FrankRuhl1924MFMediumMedium.woff" crossorigin />
         {% endif %}
         {# include current phosphor icons as css; check https://unpkg.com/phosphor-icons for latest version #}
-        <link rel="stylesheet" type="text/css" href="https://unpkg.com/phosphor-icons@1.3.2/src/css/phosphor.css" nonce="{{ reqest.csp_nonce }}"/>
+        <link rel="stylesheet" type="text/css" href="https://unpkg.com/phosphor-icons@1.4.2/src/css/icons.css" nonce="{{ reqest.csp_nonce }}"/>
```

Comment:
Speaking of Phosphor, I'm also noticing that the new icons.css has caused all of our Lighthouse tests to fail. Not sure why it's suddenly huge and taking so much time to load; perhaps pulling it in as an NPM package could help, as it won't be a 3rd party resource, though I imagine the bundle size will remain the same.

Reply:
Wow, it never occurred to me that it would break things (that seems poor on their part); we don't need the new version for any reason, I just thought I'd update since I noticed it had changed. I think we should revert this.

I was using a couple of phosphor icons for a different project and just downloaded the individual icons I wanted; we might think about doing something similar here.

Reply:
Reverted the change, but that might be something to revisit, especially if/when there are new icons only present in later Phosphor updates that we want to use.